common/src/main/java/org/apache/comet/parquet/FileReader.java

```diff
@@ -40,6 +41,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.apache.commons.compress.utils.Sets;
```

Comment:
(nit) Perhaps we could use built-in Set.

```suggestion
import java.util.Set;
```